docker/dockerfiles/artifact-ubuntu.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This docker file build the Starrocks artifacts fe & be and package them into a busybox-based image
+# Please run this command from the git repo root directory to build:
+# DOCKER_BUILDKIT=1 docker build --rm=true -f docker/dockerfiles/artifact-ubuntu.Dockerfile -t starrocks/artifact-ubuntu:tag .
+
+ARG builder=ghcr.io/dengliu/starrocks/dev-env-ubuntu:latest
```

Comment:
would you please change this to `ghcr.io/starrocks/....` so it will refer to our correct image